AGENTS.md

```
@@ -0,0 +1,296 @@
+- `design:` - UI/UX design changes
+- `fix:` - Bug fixes
+- `chore:` - Other minor changes
+- `refactor:` - Code refactoring
+- `ci:` - CI configuration changes (e.g., detekt.yml, GitHub Actions)
+- `test:` - Test code changes
+
+**Format**: `prefix: message` (lowercase prefix + colon + space + message)
+
+**Examples**:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)